chess-AI/experimental-work/NN Layout.py

```diff
@@ -1,83 +1,77 @@
-
-
-# this is the basic AI CNN layout
-# we will use this and improve it until it can be used for the final product.
-
+"""Basic layout for AI CNN architecture.
+        
+        Reference: Alpha Zero Paper pg. 13-14
+        Input convolutional layer takes (MT + L) channels of input, each channel with N * N dimensions.
+        N = 8 representing the dimensions of the chess board,
+        M = 14 representing the 12 channels of the position of each piece type and 2 channels for repetition,
+        T = 8 representing the number of past moves to consider,
+        L = 7 representing the number of constant number planes besides repetition such as castling, move count, etc.
+"""
 
 import torch
-import numpy as np
 from torch import nn
-import torch.nn.functional as F
-import torchvision
-
 
 class PlayNetwork(nn.Module):
     def __init__(self):
         super(PlayNetwork, self).__init__()
+        self.num_res_blocks = 3
         
-        # Basic layout for AI CNN, still need to polish the value head and BatchNorm2d input sizes.
-        
-        # Reference: Alpha Zero Paper pg. 13-14
-        # Input conv layer takes (MT + L) channels of input, each channel with N * N dimensions.
-        # N = 8 representing the dimensions of the chess board,
-        # M = 14 representing the 12 channels of the position of each piece type and 2 channels for repetition,
-        # T = 8 representing the number of past moves to consider,
-        # L = 7 representing the number of constant number planes besides repetition such as castling, move count, etc.
-        self.convLayer = nn.Sequential(nn.Conv2d(in_channels=119, out_channels=256, kernel_size=3, padding=1),
-                                       nn.BatchNorm2d(256),
-                                       nn.ReLU(inplace=True))
+        # Takes 119 channels of input comprised of 7 previous states' piece and repetition planes plus the current state input of 21 channels.
+        self.conv_layer = nn.Sequential(nn.Conv2d(in_channels = 119, out_channels = 256, kernel_size = 3, padding = 1, bias = False),
+                                        nn.BatchNorm2d(256),
+                                        nn.ReLU(inplace = True))
         
         # Each conv layer in a residual block uses 256 3x3 filters, padding used to keep the channel dimensions constant.
-        self.resBlock = nn.Sequential(nn.Conv2d(in_channels=256, out_channels=256, kernel_size=3, padding=1), 
-                                    nn.BatchNorm2d(256), 
-                                    nn.ReLU(inplace=True), 
-                                    nn.Conv2d(in_channels=256, out_channels=256, kernel_size=3, padding=1), 
-                                    nn.BatchNorm2d(256))
+        self.res_block = nn.Sequential(nn.Conv2d(in_channels = 256, out_channels = 256, kernel_size = 3, padding = 1, bias = False), 
```

Comment:
From style guide: "Never use spaces around `=` when passing keyword arguments or defining a default parameter value, with one exception: when a type annotation is present, do use spaces around the `=` for the default parameter value." ([link to whitespace stuff](https://google.github.io/styleguide/pyguide.html#36-whitespace)). So basically you would instead have `in_channels=256` etc like you had before and similar everywhere else.